hudi-spark/src/main/scala/org/apache/hudi/DefaultSource.scala

```diff
@@ -123,4 +127,25 @@ class DefaultSource extends RelationProvider
   }
 
   override def shortName(): String = "hudi"
+
+  private def getReadOptimizedView(sqlContext: SQLContext,
```

Comment:
we can rename to something like `getFilteredBaseFileRelation()`. Again, don't want to bring back view nomenclature into the code.

Reply:
sure, will do